src/exampleTypes/TagsQuery/index.js

```diff
@@ -19,17 +19,45 @@ let TagsQuery = ({
   popoverPosition = 'bottom right',
   popoverArrow,
   popoverOffsetY,
-  theme: { Icon, TagsInput, Tag, Popover },
+  Loader,
+  theme: { Icon, TagsInput, Tag, Popover, Loader: ThemeLoader },
   joinOptions,
   ...props
 }) => {
+  Loader = Loader || ThemeLoader || StripedLoader
```

Comment:
these shouldn't be necessary - the contexture react theme API allows you to override the theme components, so we should just use that